api/orchestrator/orchestrator.proto

```diff
@@ -346,4 +376,42 @@ message Requirement {
   string description = 3;
 
   repeated string metric_ids = 4;
+}
+
+message GetCertificateRequest {
+  string certificate_id = 1;
+}
+
+message ListCertificatesRequest {}
+message ListCertificatesResponse { repeated Certificate certificates = 1; }
+
+message UpdateCertificateRequest {
+  string certificate_id = 1;
+	Certificate certificate = 2;
+}
+
+message CreateCertificateRequest { Certificate certificate = 1; }
+
+message RemoveCertificateRequest { string certificate_id = 1; }
+
+message Certificate {
+  string ID = 1;
```

Comment:
This should be `id` or is there then a problem with Gorm persistance?

Reply:
I used ID because that is what gorm uses as the default primary key; I didn't try id though

Reply:
You will never get `ID` past mr. protobuf style guide @lebogg anyway :D

Reply:
Please don't do gitblame on that line :laughing:
`id` seems to work. I will push it.